datafusion/common/Cargo.toml

```diff
@@ -39,12 +39,12 @@ pyarrow = ["pyo3"]
 
 [dependencies]
 apache-avro = { version = "0.14", features = ["snappy"], optional = true }
-arrow = { version = "21.0.0", features = ["prettyprint"] }
+arrow = { version = "22.0.0", features = ["prettyprint"] }
 avro-rs = { version = "0.13", features = ["snappy"], optional = true }
 cranelift-module = { version = "0.87.0", optional = true }
 object_store = { version = "0.4.0", optional = true }
 ordered-float = "3.0"
-parquet = { version = "21.0.0", optional = true, features = ["arrow"] }
-pyo3 = { version = "0.16", optional = true }
+parquet = { features = ["arrow"], optional = true, version = "22.0.0" }
```

Comment:
It seems to be a common practice to always have the version field at first

Reply:
This was done by an automated tool. I'll fix my tool and re-submit.

Reply:
Resolved with this ordering: https://github.com/avantgardnerio/workspace-gen/blob/891699e8cd4d85c7349cf70518a1f18628ea570d/src/main.rs#L242